blockreassurance.php

```diff
@@ -307,10 +307,10 @@ public function getContent()
 
         $this->context->smarty->assign(array(
             'addons_category' => $categoryFetcher,
-            'psr_hook_header' => Configuration::get('PSR_HOOK_HEADER'),
-            'psr_hook_footer' => Configuration::get('PSR_HOOK_FOOTER'),
-            'psr_hook_product' => Configuration::get('PSR_HOOK_PRODUCT'),
-            'psr_hook_checkout' => Configuration::get('PSR_HOOK_CHECKOUT'),
+            'psr_hook_header' => (int) Configuration::get('PSR_HOOK_HEADER'),
+            'psr_hook_footer' => (int) Configuration::get('PSR_HOOK_FOOTER'),
+            'psr_hook_product' => (int) Configuration::get('PSR_HOOK_PRODUCT'),
+            'psr_hook_checkout' => (int) Configuration::get('PSR_HOOK_CHECKOUT'),
```

Comment:
```suggestion
            'psr_hook_checkout' => Configuration::getInt('PSR_HOOK_CHECKOUT'),
```

Reply:
:ok_hand:

Reply:
So you were ok with all the modifs but made none of them? Maybe you got the wrong emoji? 😅

Reply:
@jolelievre It's ever done ;)